### DISCUSSION TITLE
disable TNT outside claims

### DISCUSSION THREAD

**Discussion by DailyStruggle** [GriefPrevention/GriefPrevention#554] [Q&A] [answered]
As a player, I would like to use TNT and charged creepers for my builds. 

As an operator, I want to minimize griefing as much as possible.

Is there a way to disable explosion block damage globally except in claimed areas with claimexplosions enabled?

**Comment by RoboMWM** [comment]
Interesting, you kind of have a UML-esque story phrasing here. I like it, just expand on it a bit more c:

By default the config stops explosions that are above sea level. You can reduce the sea level manually in the config.

**Comment by DailyStruggle** [comment, marked as answer]
do you mean 'SeaLevelOverrides'?

does it currently affect all blocks or only surface blocks?

edit- -1 uses default sea level. I needed to manually set the value
